Packs/Vectra/Integrations/VectraEventCollector/test_data/search_detections.json

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
No need for such a long file let's shrink it as minimal as possible so it will give you what you need